pkg/cluster/cluster.go

```diff
@@ -437,6 +450,30 @@ func createClusterSpec(config Spec, awsClient aws.Client) (*cmv1.Cluster, error)
 		return nil, fmt.Errorf("Custom properties key %s collides with a property needed by rosa", properties.CLIVersion)
 	}
 
+	/**
+	1) Poll the cluster with same arn from ocm
+	2) Check the status and if pending enter to a loop until it becomes installing
+	*/
+
+	deadline := time.Now().Add(5 * time.Minute)
+	for {
+		pendingCluster, err := GetPendingClusterForARN(ocmClusterClient, awsCreator.ARN)
+		if err != nil {
+			reporter.Errorf("Error getting cluster using ARN '%s'", awsCreator.ARN)
+			os.Exit(1)
+		}
+		if time.Now().After(deadline) {
+			reporter.Errorf("Timeout waiting for the cluster '%s' installation", pendingCluster.ID())
+			os.Exit(1)
+		}
+		if pendingCluster == nil {
+			break
+		} else {
+			reporter.Infof("Waiting for the cluster '%s' in the same User ARN to start installing", pendingCluster.ID())
```

Comment:
Change this to `Waiting for cluster '%s' with the same creator ARN to start installing`